### DISCUSSION TITLE
Reducing a bit the amount of an existing ore

### DISCUSSION THREAD

**Discussion by CroaBeast** [DerFrZocker/Ore-Control#15] [Q&A] [answered]
I just want to know how can I reduce the amount or maybe the size of a vein of an existing ore. Thanks

**Comment by DerFrZocker** [comment, marked as answer]
You can simply select the ore you want to change, then go to the generator settings and select the size setting. And reduce it accordingly. Note, that not all ores / features such as Glowstone supported the size setting.

**Reply by CroaBeast** [reply]
Thanks, I'll try that!
